packages/bot-web-ui/src/utils/journal-notifications.js

```diff
@@ -0,0 +1,62 @@
+import { localize } from '@deriv/translations';
+import { messageWithButton, arrayAsMessage } from '../components/notify-item.jsx';
+
+const showErrorMessageWithButton = (message, block_id, showErrorMessage, centerAndHighlightBlock) => {
+    showErrorMessage(
+        messageWithButton({
+            unique_id: block_id,
+            type: 'error',
+            message,
+            btn_text: localize('Go to block'),
+            onClick: () => {
+                centerAndHighlightBlock();
+            },
+        })
+    );
+};
+
+export const validateJournalMessage = (
```

Comment:
Since this returns a boolean, could we name it as such? e.g. `isValidJournalMessage`

Reply:
Since this is checking for conditions to show custom journal message components, would something like `isCustomJournalMessage` be more appropriate?